### ISSUE TITLE
jetty 11 mismatch version of setuid jar

### ISSUE THREAD

**Issue by simonlee** [eclipse/jetty.project#5904]
The included setuid jar file:
```
lib/setuid/jetty-setuid-java-1.0.3.jar
```
does not match the jar referenced in `modules/setuid.mod`:
```
[lib]
lib/setuid/jetty-setuid-java-1.0.4.jar
```

**Comment by olamy** [comment]
fixed with https://github.com/eclipse/jetty.project/commit/5b7472f616ebd23efe02ba658dc1b1e5b4f0e3e7#diff-c14d587c50eced6d1cbad346cd86e3efebbefa200848f52bb813b388b5c65cce

what jetty 11 version are you using?

**Comment by simonlee** [comment]
I am using jetty 11.0.0

11.0.0 tgz.sha1 06bb0ad49d9c18b21053ff5f34d8a55e28d4f606

**Comment by olamy** [comment]
Thanks for the report.
But it's already fixed in jetty-11.0.x branch and will be part of next release.
